services/ui-src/src/libs/detailLib.ts

```diff
@@ -101,7 +101,7 @@ export const defaultDetail: OneMACDetail = {
     },
   ],
   actionsByStatus: Workflow.defaultActionsByStatus,
-  show90thDayInfo: true,
+  show90thDayInfo: false,
```

Comment:
Based on the hmmm-ing and haw-ing at refinement over if they would ever bring this back, I agree it is better to disable instead of removing.